app/views/posts/new.html.erb

```diff
@@ -1,3 +1,12 @@
+<link href="https://fonts.googleapis.com/css?family=Tenor+Sans&display=swap" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="base.css">
+
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#create">Create</a>
+<a href="#friends">Friends</a>
+</div>
+
```

Comment:
We could consider putting these in a template for all pages. Elfi might have an idea how to do this